airflow/bin/cli.py

```diff
@@ -251,6 +261,7 @@ def pools(args):
     with open(args.filepath, 'r') as stream:
         try:
             pools_config = yaml.load(stream)
+            pools_config = yaml.load(stream)
```

Comment:
Copy paste the wrong thing!

Reply:
thanks. I didn't change this line. will remove it.